components/ProductMatrix.vue

```diff
@@ -21,15 +24,17 @@
 <script lang="ts">
 import Vue from 'vue'
 import { mapGetters, mapActions } from 'vuex'
+import { vueWindowSizeMixin } from 'vue-window-size'
```

Comment:
Can one trust this source? Should code review or something else be done when including external sources like this?